packages/iris-grid/src/sidebar/visibility-ordering-builder/VisibilityOrderingBuilder.tsx

```diff
@@ -1049,6 +1109,14 @@ class VisibilityOrderingBuilder extends Component<
       treeItems
     );
 
+    const queryParams = {
```

Comment:
i thought this would be nicer to pass in to SearchInput rather than creating two new optional props. Could revert to that if preferred though